Magento_Theme/templates/html/pager.phtml

```diff
@@ -108,18 +104,18 @@
                     <?php endif; ?>
 
                     <?php if ($block->canShowLast()): ?>
-                    <li class="pager__item">
-                        <a class="pager__link link" href="<?php /* @escapeNotVerified */ echo $block->getLastPageUrl() ?>">
-                            <?php /* @escapeNotVerified */ echo $block->getLastPageNum() ?>
-                        </a>
-                    </li>
+                        <li class="pager__item">
+                            <a class="pager__link link" href="<?php /* @escapeNotVerified */ echo $block->getLastPageUrl() ?>">
+                                <?php /* @escapeNotVerified */ echo $block->getLastPageNum() ?>
+                            </a>
+                        </li>
                     <?php endif; ?>
 
                     <?php if (!$block->isLastPage()): ?>
                         <li class="pager__item">
                             <?php $text = $block->getAnchorTextForNext() ? $block->getAnchorTextForNext() : '';?>
-                            <a class="<?php /* @escapeNotVerified */ echo $text ? 'link ' : 'action '?> pager__link next" 
-                               href="<?php /* @escapeNotVerified */ echo $block->getNextPageUrl() ?>" 
+                            <a class="<?php /* @escapeNotVerified */ echo $text ? 'link ' : 'action '?> pager__link next"
```

Comment:
attrs to new line and ise shorhand instead of echo

Reply:
splited and added shorhand